src/Service/Generate/CreateService.php

```diff
@@ -32,20 +32,20 @@ final class CreateService
     private Connection $db;
     private ConsoleHelper $consoleHelper;
     private MigrationService $migrationService;
-    private View $view;
+    private WebView $webView;
```

Comment:
Why changing it?

Reply:
In the application we define webview, no view, simple configuration.